ultimatejobweb/cron.py

```diff
@@ -0,0 +1,5 @@
+import os
+
+
+def my_scheduled_job():
+    os.system('echo hello >> /vagrant/cron.out')
```

Comment:
so this PR has two PRs 
One is the cron, which is not running any script. 
2nd is adding RH and tests, I think you should split PRs.